examples/cluster-go/go.mod

```diff
@@ -0,0 +1,10 @@
+module cluster-go
+
+go 1.14
+
+require (
+	github.com/pulumi/pulumi-eks/sdk v0.21.0 // indirect
+	github.com/pulumi/pulumi/sdk/v2 v2.18.2
+)
+
+replace github.com/pulumi/pulumi-eks/sdk => ../../sdk
```

Comment:
I'm pretty sure this won't run in CI as it creates a temp GOPATH and it won't be able to traverse the ../../ notation

Reply:
> I'm pretty sure this won't run in CI as it creates a temp GOPATH and it won't be able to traverse the ../../ notation

The reason for this to exist currently is because we don't actually have `github.com/pulumi/pulumi-eks/sdk` - this change adds it. I added a TODO. I could also exclude the example and add it in a separate PR. Let me know if you have a strong preference either way.

Reply:
TODO works for me. 👍